src/components/DFProfile.svelte

```diff
@@ -118,6 +118,7 @@
                             containerWidth={profileWidth}
                             view={previewView}
                             totalRows={dataframeProfile?.shape?.[0]}
+                            isIndex={i === 0}
```

Comment:
I dont love using the column order to determine if the column is an index, is there a more stable way to do this? This will be fragile especially once we add options to toggle index visualizations on or off since the 1st column wont always be the index

Reply:
The new PR doesn't use i === 0 as the identifier of index right now.